crates/frontend/src/pages/function.rs

```diff
@@ -89,7 +89,9 @@ pub fn function_page() -> impl IntoView {
                         let (function_rs, _) = create_signal(function);
                         publish_error_ws.set("".to_string());
                         match function_rs.get().published_at {
-                            Some(val) => show_publish_ws.set(val < function_rs.get().draft_edited_at),
+                            Some(val) => {
+                                show_publish_ws.set(val < function_rs.get().draft_edited_at)
```

Comment:
We don't need the brackets here, right?

Reply:
i just formatted, sure will remove this.